suite.go

```diff
@@ -95,3 +103,45 @@ func (s *Suite) Reset(ctx context.Context) {
 		s.network.reset(ctx)
 	}
 }
+
+// Start starts the suite. This starts all networks in the suite and the underlying containers,
+// as well as the daemon logger, if Verbosity is enabled.
+func (s *Suite) Start(ctx context.Context) {
+	if logWatcher == nil {
+		printf("(daemon) starting logging")
+		logWatcher = logger.NewLogWatcher()
+		go func() {
+		InfLoop:
+			for {
+				select {
+				case <-logDone:
+					break InfLoop
+				case msg := <-logWatcher.Msg:
+					printf("(daemon) %s", msg.Line)
+				case err := <-logWatcher.Err:
+					printf("(d err ) %s", err)
+				}
+			}
+			printf("(daemon) stopping logging")
+			logWatcher.Close()
+		}()
+	}
+
+	if s.network != nil {
+		s.network.start(ctx)
+	}
+}
+
+// Stop stops the suites. This stops all networks in the suite and the underlying containers,
+// as well as the daemon logger.
+func (s *Suite) Stop() error {
```

Comment:
I would call it `Close` even if it sound stupid. Its common interface. Would be quite handy.